docs/reference/tab-widgets/troubleshooting/data/migrate-to-data-tiers-routing-guide.asciidoc

```diff
@@ -0,0 +1,224 @@
+// tag::cloud[]
+In order to get the shards assigned we need to call the 
+<<ilm-migrate-to-data-tiers, migrate to data tiers routing>> API which will
+resolve the conflicting routing configurations towards using the standardized
+<<data-tiers, data tiers>>. This will also future-proof the system by migrating 
+the index templates and ILM policies if needed.
+
+**Use {kib}**
+
+//tag::kibana-api-ex[]
+. Log in to the {ess-console}[{ecloud} console].
++
+
+. On the **Elasticsearch Service** panel, click the name of your deployment. 
++
+
+NOTE:
+If the name of your deployment is disabled your {kib} instances might be
+unhealthy, in which case please contact https://support.elastic.co[Elastic Support],
+or your deployment doesn't include {kib}, in which case all you need to do is 
```

Comment:
Run on sentence: Should split the sentence into two. End the first after "please contact Support.", Start the new sentence with "If your deployment doesn't include Kibana, then all you need to do ...."

Reply:
Yes, good spot. Will do this in a follow-up for all guides. Thanks for the suggestion